benchmark/catalyst_benchmark/chemvqe_catalyst.py

```diff
@@ -0,0 +1,97 @@
+import pennylane as qml
+import jax.numpy as jnp
+import catalyst
+import numpy as np
+
+from typing import Any, Dict
+from dataclasses import dataclass
+from dataclasses_json import dataclass_json
+from pennylane import AllSinglesDoubles
+from catalyst import qjit
+from jax.core import ShapedArray
+
+from .types import Problem
```

Comment:
I was having issues running these modules as scripts because of the relative import here, getting:
```pytb
ImportError: attempted relative import with no known parent package
```
Maybe it's best to just use an absolute import, and also rename the module so the name doesn't conflict with the builtin `types` module? For example:
```suggestion
from benchmark_classes import Problem
```
or something similar.

Reply:
OK, I see, entry-point Python scripts don't fit well into Python modules. Let me suggest another solution: In the latest version of this PR I have moved the entry point part of the script outside the module folder. Now there are `benchmark.py` and `runbatch.py` top-level scripts for performing individual measurements and for running the whole measurement cycle. Does it solve your problem?

Reply:
Generally I find it useful to be able to just add something like

```py
if __name__ == "__main__":
    # call relevant function
```

and hit F5 to run and debug certain modules. I was able to do that by fixing the imports as suggested above.

Is there a disadvantage to using absolute imports over relative ones in the benchmark suite?

Reply:
Do you run the script as `python ./script.py` or as `python -m script` ? I believe Python treats the script as a part a module only in the latter case.

> Is there a disadvantage to using absolute imports over relative ones in the benchmark suite?

I think that code portability would decrease if we use absolute imports here. Currently one can easily move the module folder across the filesystem if they want (and I think we might want to do it in future). The sources do not require any fixing. If we use absolute imports, certain fixes would be required before moving. Let me know if you still think that it would be better to use absolutes, I am ready to change this.

Reply:
> I think that code portability would decrease if we use absolute imports here. Currently one can easily move the module folder across the filesystem if they want (and I think we might want to do it in future). The sources do not require any fixing. If we use absolute imports, certain fixes would be required before moving. Let me know if you still think that it would be better to use absolutes, I am ready to change this.

Ah you're right, actually my current suggestion would break running it from the top-level scripts. What I was thinking of was an absolute path starting with the package name, which is how we handle imports in the catalyst package as well:

So `from .types import` would turn into `from catalyst_benchmark.types import`.

- This still works with the top-level scripts and the code is still portable (unless you change the location of `catalyst_benchmark/types.py` itself.
- It does allow running all source files as scripts as long as the parent package (`catalyst_benchmark`) is on the python path.